src/fides/api/ops/schemas/messaging/messaging_secrets_docs_only.py

```diff
@@ -0,0 +1,20 @@
+from typing import Union
+
+from fides.api.ops.schemas.base_class import NoValidationSchema
+from fides.api.ops.schemas.messaging.messaging import (
+    MessagingServiceSecretsMailgun,
+    MessagingServiceSecretsTwilioSMS,
+)
+
+
+class MessagingSecretsMailgunDocs(MessagingServiceSecretsMailgun, NoValidationSchema):
+    """The secrets required to connect to Mailgun, for documentation"""
+
+
+class MessagingSecretsTwilioDocs(MessagingServiceSecretsTwilioSMS, NoValidationSchema):
```

Comment:
Is there also an option for Twilio email?

Reply:
I think I missed this one, good catch!